testdata/p4_16_errors_outputs/issue1803_same_table_name.p4-stderr

```diff
@@ -0,0 +1,2 @@
+[--Werror=legacy] error: Name 't0' is used for multiple objects in the P4Info message
```

Comment:
It would be nice to improve this error by displaying the P4 constructs that have the name in the source code. Do you have references to the IR nodes at this point?

Reply:
We don't have the IR nodes at this point